app/views/users/show.html.erb

```diff
@@ -1 +1,29 @@
-<h1>List of all users by id</h1>
\ No newline at end of file
+<div class="main_container">
+<div class="profile">
+<div class="photos">
+    <ul class="photo"> 
+   <li> <img src="<%= @users.photo %>"> </li>
+   </ul>
+</div>
+<div class="names_posts">
+    <ul class="name_post"> 
+   <li class="list_name"> <%= @users.name %> </li>
```

Comment:
- [ ] Kindly take note that for good user experience you ought to use the `link `element that will redirects users to the `users_post `page. 👍 I was only able to access the page by typing the `url `manually. ℹ️ 

- [ ] Kindly ensure that `three most recent posts` are displayed by default according to the project requirements. 👍